br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -349,7 +341,8 @@ func (local *local) SplitAndScatterRegionByRanges(
 			zap.Int("skipped_keys", skippedKeys),
 			zap.Int("scatterCount", scatterCount),
 			zap.Int("regions", len(scatterRegions)),
-			zap.Duration("take", time.Since(startTime)))
+			zap.Duration("take", time.Since(startTime)),
+			zap.Error(err))
 	}
 	return nil
```

Comment:
Do we need to handle this error?

Reply:
Emm, it's hard to say. Until now, we didn't treat scatter region error as a critical error.